src/modules/createTask.js

```diff
@@ -0,0 +1,35 @@
+import {
+  retrieveLocalStorage, saveLocalStorage, taskObj, isEmpty,
+} from './Utils.js';
+import {
+  insertTaskInDom,
+  loadTasksEventHandlers,
+} from './renderTask.js';
+
+export const createTask = (inputDesc) => {
+  const inputTaskDescription = inputDesc.value;
+  const tasksList = retrieveLocalStorage('tasks-list');
+  if (!isEmpty(inputTaskDescription)) {
+    const id = tasksList.length + 1;
+    const newTask = taskObj(inputTaskDescription, false, id);
+    tasksList.push(newTask);
+    inputDesc.value = '';
+    inputDesc.focus();
+    saveLocalStorage('tasks-list', JSON.stringify(tasksList));
+    insertTaskInDom(newTask);
+    loadTasksEventHandlers();
+  }
+};
+
+export const createTaskEventHandler = () => {
+  const inputDescription = document.querySelector('#inputTaskDescription');
+  const btnAddTask = document.querySelector('#btnAddTask');
+  inputDescription.addEventListener('keydown', (event) => {
+    if (event.code === 'Enter') {
+      createTask(inputDescription);
+    }
+  });
+  btnAddTask.addEventListener('click', () => {
+    createTask(inputDescription);
+  });
+};
```

Comment:
📍 [Optional] Please keep in mind that if you use a `keydown` event the action will ocurr every time the user press a key so I suggest you use a `timer` to delete the number of times that the evet will be triggered or also you can change the event to `focusout`

Reply:
@AndreaM2429 It is an excellent suggestion. Thank you so much. I'm going to try using a `timer` to improve the performance of this functionality.